### ISSUE TITLE
--skip-plugins flag is permanently deactivate all plugins

### ISSUE THREAD

**Issue by halilemreozen** [wp-cli/wp-cli#4943]
Hi Everyone,

It's the first issue of mine in this project, hope it's not opened to wrong place :)

## TL;DR;
When I use --skip-plugins, all plugins are disabled on related WordPress site.

# Background Story :

I have a management panel for my customers and it uses wp-cli features as possible,

As you know if there is a problem occurred on plugin or themes, wp-cli process stuck there and that's for wp-cli offers --skip-plugins and --skip-themes to skip these loading steps.

Yesterday I add these flags to an operation and after that, 4 site owner (different servers) report a bug about deactivated plugins.

After that I tried the following command on 4 different servers with different wp-cli versions and sites, the result is same at all of them.

## Steps to reproduce:

Run following command : 

‘wp theme list  --skip-plugins --skip-themes  --allow-root --path=/home/hosting_user/www/’

Observe Error: ‘All plugins are disabled.’

# Details : 
## What is your system?
Centos 7
Linux 3.10.0-714.10.2.lve1.5.17.1.el7.x86_64 GNU/Linux
## Which version of PHP are you using?
PHP 5.6.37 
## Which version of WordPress are you using?
$wp_version = '4.9.8';
## Can you share the results of which wp?
Tested with :
WP-CLI 1.5.0
WP-CLI 2.0.1

**Comment by halilemreozen** [comment]
Hi @schlessera  ,

When I use the `--skip-plugins` flag, yes all of my plugins are being deactivated until I manually re-activate them, when I used this flag I was expecting just skip the plugin load stage only for that wp-cli command.

Is it should works like `wp plugin deactivate --all` ?

**Comment by danielbachhuber** [comment]
Hi @halilemreozen,

This issue seems specific to your environment, not a bug with WP-CLI itself. Feel free to pop into the `#cli` channel on Slack if you'd like assistance debugging.
